.github/workflows/CI.yml

```diff
@@ -143,17 +143,21 @@ jobs:
         - tracing-tower
         - tracing-opentelemetry
         - tracing
-        - tracing-subscriber
```

Comment:
i believe this means we are no longer running the `-Z minimal-versions` check for `tracing-subscriber` on stable --- we should probably keep the crate here, and _just_ add the `exclude`.

Reply:
Yes, the change you've made there will run this check for `tracing-subscriber` on Rust 1.50.0. However, the build matrix for this job will run the checks for all the crates in the `subcrate` list for _all_ the toolchains in the `toolchain` list, which is the MSRV version _and_ stable:
https://github.com/tokio-rs/tracing/blob/198694ffa8a50d6970dd02c4edc286501c996e9f/.github/workflows/CI.yml#L146-L148

By removing `tracing-subscriber` from the build matrix `subcrate` list, this change also removes the check run on stable, which is nice, since it ensures that the crates also build with minimal dependency versions on the current stable Rust toolchain. The `exclude` will still remove the build with 1.49.0 for `tracing-subscriber`, and the `include` will add a build with `tracing-subscriber` on 1.50.0, even if we don't remove `tracing-subscriber` from the subcrate list.

I know this change is based on the way the `tracing-appender` builds are declared, but I think the builds for `tracing-appender` here are not quite right: it _should_ be included in the `subcrate` list, and just have an `exclude` key to remove the build for that crate on 1.49.0.

This isn't a major problem, but it's nice to also do `-Zminimal-versions` builds on stable Rust as well.

Reply:
Done, I added `tracing-appender` too